src/app/game/game-draw/game-draw.component.ts

```diff
@@ -14,20 +21,33 @@ import { GameConfigService } from '../services/game-config.service';
 import { TranslationService } from '@/app/core/translation.service';
 import { TranslatePipe } from '@/app/core/translation.pipe';
 import { GameStateService } from '../services/game-state-service';
+import { CustomColorsIO } from '@/app/shared/customColors';
+import { SpeechBubbleComponent } from '../speech-bubble/speech-bubble.component';
+import { OAvatarComponent } from '@/assets/avatars/o-avatar/o-avatar.component';
+import { IAvatarComponent } from '@/assets/avatars/i-avatar/i-avatar.component';
+import { ViewChild } from '@angular/core';
 
 @Component({
   selector: 'app-drawing',
   templateUrl: './game-draw.component.html',
   styleUrls: ['./game-draw.component.scss'],
   standalone: true,
-  imports: [MatIcon, MatIconButton, UpperCasePipe, TranslatePipe],
+  imports: [
+    MatIcon,
+    MatIconButton,
+    UpperCasePipe,
+    TranslatePipe,
+    SpeechBubbleComponent,
+    IAvatarComponent,
+    OAvatarComponent,
+  ],
 })
 export class GameDrawComponent implements OnInit, OnDestroy {
   config!: GameLevelConfig;
   subscriptions = new Subscription();
 
-  canvas = viewChild.required<ElementRef<HTMLCanvasElement>>('canvas');
-  countDown = viewChild.required<ElementRef<HTMLSpanElement>>('countDown');
+  @ViewChild('canvas', { static: true }) canvas!: ElementRef<HTMLCanvasElement>;
+  @ViewChild('countDown', { static: true }) countDown!: ElementRef<HTMLSpanElement>;

```

Comment:
Can this eventually be done with question marks instead? Me no like exclamation marks 🥴